### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by danipoak**
Updated README with more descriptive instructions. Outlined two methods, `--daemon` and an alternative using crontab every 30 minutes.

### CODE REVIEW

**Comment by cameronaaron** [comment]
[089fdfd](https://github.com/beeper/pypush/pull/43/commits/089fdfdc7bc65e956316c68cdffbd8398b78ab87) needs to be reverted as it would break the code

**Comment by JJTech0130** [comment]
Why? It should work in the latest commit

**Comment by danipoak** [comment]
> [089fdfd](https://github.com/beeper/pypush/pull/43/commits/089fdfdc7bc65e956316c68cdffbd8398b78ab87) needs to be reverted as it would break the code

I'm not seeing what the issue is here. It's not changing any code. And this is the correct way to run `demo.py` with the daemon.

**Comment by danipoak** [comment]
Fixed a small typo with cron and Screen.

**Comment by danipoak** [comment]
Added `sleep 60` which accounts for some systems that are slower to startup which results in screen or demo.py failures due to services that haven't started up yet.

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ However, Beeper is completely free and easy to use, and comes packed with multip
 You will first install Pypush onto your machine. *Please keep in mind that you will have to have a script running on a home server or PC 24/7 to keep your number active! See below for instructions.*
 
 ### PNRgateway
-In order for Apple to verify your number, a specialized message has to be sent from your phone to Apple's "gateway number" and have the response captured. This number is different for each carrier, however the newest app version should automatically find your gateway number. If PNRgateway cannot find your gateway number, see below for help.
+In order for Apple to verify your number, a specialized message has to be sent from your phone to Apple's "gateway number" and have the r     nbnnnnnnnnnnnnnnnnnnnnnnesponse captured. This number is different for each carrier, however the newest app version should automatically find your gateway number. If PNRgateway cannot find your gateway number, see below for help.
```
Comment (niggowai) [inline README.md]: Is this difference intended?

Comment (danipoak) [reply README.md]: Nope. Fixing.

**Comment by danipoak** [comment]
Fixed repeating character typo.

**Comment by danipoak** [comment]
Lots of changes: added `--cronreg` as an alternative way to check when certificate expires, if it's within an hour it reregisters, if not it disconnects
There is a working powershell script to automate installation/activation for Windows 10/11

**Comment by SpaceSaver** [comment]
> Lots of changes: added `--cronreg` as an alternative way to check when certificate expires, if it's within an hour it reregisters, if not it disconnects There is a working powershell script to automate installation/activation for Windows 10/11

I think an hour is too long, you may not get the extension if dereg is still an hour out.

**Comment by danipoak** [comment]
> > Lots of changes: added `--cronreg` as an alternative way to check when certificate expires, if it's within an hour it reregisters, if not it disconnects There is a working powershell script to automate installation/activation for Windows 10/11
> 
> I think an hour is too long, you may not get the extension if dereg is still an hour out.

I was worried about that but I've been testing it and it's gone from 30 minutes to 5 days since I started it on Sunday.

**Pull request merged**